lib/sender.js

```diff
@@ -67,67 +68,91 @@ module.exports = {
 */
 function send(client) {
     return function send(message, options) {
-        return new Promise((resolve, reject) => {
-            options = options || {};
+        const body = composeSenderRequestBoddyForOneMessage(message, options);
+        return makeRequest(client, body);
+    };
+}
+
+/**
+    The sendAll Method for multiple messages
+    @param {Array}  messages - An array of objects containing the fields described in send Method [{message, options}, ... ]
+*/
+function sendAll(client) {
```

Comment:
Let's call it batch, that's also reflecting the name of the endoint 

I guess a proper overload of `push` is not possible, or would lead to if/else :spaghetti:  ?